### ISSUE TITLE
BFA artifact power bar

### ISSUE THREAD

**Issue by robwoodok** [fizzwidget/faction-friend#55]
When you have a toon that is pre BFA event, the addon works as it always has:  it splits the XP bar into XP and Rep on top.    When you do the BFA event, the Game splits the XP bar into Artifact Power on top and XP below, then your addon replaces the XP half bar with rep.

so...is there anyway to program to replace the Artifact Power half bar with rep?

**Comment by ShaunaGordon** [comment]
Yeah, this is a known issue (see #19 ) dating into Legion. To my knowledge, a solution hasn't yet been implemented.

**Comment by fizzwidget** [comment]
https://github.com/fizzwidget/faction-friend/releases/tag/v12.0 includes a nearly full rewrite of this addon, with a few new features, few old features retired, and new implementations throughout — so any existing issues are probably either resolved or irrelevant. Please file new issues if you see old problems recurring or new problems arising!

For this in particular — sometime since Legion (probably the big UI rewrite in Dragonflight?), the base UI changed its XP/reputation/etc bars to be more modular, so the new FactionFriend hooks into that and seems to work okay with all the places where the base UI switches bars. Doesn't look like there's a good way to force the UI to show reputation when it wants the two bars to be something else, though.
